test/kind/basic_test.rb

```diff
@@ -261,7 +261,14 @@ def test_Kind_assert_hash!
     assert_raises_with_message(
       ArgumentError,
       ':keys or :schema is missing'
-    ) { Kind.assert_hash!({}, some: :arg) }
+    ) { Kind.assert_hash!({a: 'a'}, some: :arg) }
```

Comment:
@haroldofurtado thanks for your contribution. But this change is not necessary, because you can perform the keywords before the hash presence validation. e.g:
```ruby
raise ArgumentError, ':keys or :schema is missing' if !options.key?(:keys) || !options.key?(:schema)
raise ArgumentError, "hash cannot be empty" if hash.empty?
```